modules/gdscript/tests/test_completion.h

```diff
@@ -161,6 +161,7 @@ static void test_directory(const String &p_dir) {
 				owner = scene->get_node(conf.get_value("input", "node_path", "."));
 			}
 
+			ERR_PRINT_OFF;
```

Comment:
We probably want a comment to justify why we're silencing errors here, as it's not immediately obvious form the code that this is expected to print errors and we don't mind.